templates/aai-secret.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "sda.fullname" . }}-aai
+type: Opaque
+data:
+  id:
+  secret:
```

Comment:
This should probably be renamed to `metadata-secrets` or something since it needs its own set of mongodb credentials

Reply:
Instead of renaming this, I renamed mongo secret to db secret, which is created even if mongo deployment is not created. Is this be a sufficient change?